src/pages/workspace/downgrade/DowngradeIntro.tsx

```diff
@@ -67,14 +70,26 @@ function DowngradeIntro({onDowngrade, buttonDisabled, loading}: Props) {
                     <Text style={[styles.textBold, styles.textSupporting]}>{translate('workspace.downgrade.commonFeatures.benefits.warning')}</Text>
                 </Text>
             </View>
-            <Button
-                isLoading={loading}
-                text={translate('common.downgradeWorkspace')}
-                success
-                onPress={onDowngrade}
-                isDisabled={buttonDisabled}
-                large
-            />
+            {policyID ? (
+                <Button
+                    isLoading={loading}
+                    text={translate('common.downgradeWorkspace')}
+                    success
+                    onPress={onDowngrade}
+                    isDisabled={buttonDisabled}
+                    large
+                />
+            ) : (
+                <Button
+                    text={translate('workspace.common.goToWorkspaces')}
+                    success
+                    onPress={() => {
+                        Navigation.goBack();
+                        Navigation.navigate(ROUTES.SETTINGS_WORKSPACES);
```

Comment:
```suggestion
                        Navigation.goBack();
                        Navigation.navigate(ROUTES.SETTINGS_WORKSPACES);
```

Let's use 
`Navigation.navigate(ROUTES.SETTINGS_WORKSPACES, CONST.NAVIGATION.TYPE.UP);`

to replace the current screen